stock_available/models/product_template.py

```diff
@@ -10,6 +10,27 @@ class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
     @api.multi
+    def _product_available(self, name=None, arg=False):
+        res = super(ProductTemplate, self)._product_available(name, arg)
+
+        variants = self.env['product.product']
+        for product in self:
+            variants += product.product_variant_ids
```

Comment:
Just curious, any particular reason not to use 
``
variants = self.mapped('product_variant_ids')
```
 to replace these three lines?